sql/src/main/java/org/apache/apex/malhar/sql/planner/RelInfo.java

```diff
@@ -0,0 +1,119 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.apex.malhar.sql.planner;
+
+import java.util.List;
+
+import org.apache.calcite.rel.type.RelDataType;
+
+import com.datatorrent.api.Operator;
+
+/**
+ * This object communicates stream and connection data between various stages of relational algebra.
+ */
+public class RelInfo
```

Comment:
That's why RelInfo... Rel here stands for "relation"... A relation is single or composite. I think RelInfo looks fine.